pac4j-core/src/main/java/org/pac4j/core/client/IndirectClient.java

```diff
@@ -154,12 +154,32 @@ public void setCallbackUrl(final String callbackUrl) {
 
     public String getCallbackUrl() { return this.callbackUrl; }
 
-    public CallbackUrlResolver getCallbackUrlResolver() {
-        return callbackUrlResolver;
+    /**
+     * Use {@link #getUrlResolver()} instead.
+     *
+     * @return the URL resolver for the callback URL
+     */
+    @Deprecated
```

Comment:
Based on what I see on the mailing list, the `CallbackUrlResolver` concept is used by many people. These deprecated methods are meant to ease the transition to the new `UrlResolver` concept...